src/tx/builder/schema.ts

```diff
@@ -292,11 +327,27 @@ export const TX_FEE_BASE_GAS = (txType) => {
     [TX_TYPE.gaAttach]: 5,
     [TX_TYPE.gaMeta]: 5,
     [TX_TYPE.payingFor]: 1 / 5
-  })[txType] || 1
+  }
```

Comment:
```suggestion
  } as const
```
and remove `[key in TxType]?: number` from the above